building/upstream.log

```diff
@@ -34,3 +34,24 @@
   comment: mostly matches manual git-archive export, except for a couple of minor versioning differences, as expected
 
   compress: gunzip, xz
+
+- output: rkt-1.29.0.tar.xz
+  last: 2018-01-31
+
+  git: https://github.com/rkt/rkt
+  tag: v1.29.0
+  compress: xz
```

Comment:
I managed to reproduce this one, but a line saying `git archive v1.29.0 --prefix='rkt-1.29.0/'` would have been a useful hint.